docs/release.md

```diff
@@ -75,35 +75,54 @@ function of the appropriate pallets.
 
 ### Extrinsic Ordering & Storage
 
-Offline signing libraries depend on a consistent API of call indices and functions. Nowadays, we are no longer using
-indices but the task of checking whether the API remains compatible remained and the name "Extrinsic Ordering" remained
-as well although we no longer check the ordering itself.
-
-Compare the metadata of the current and new runtimes and ensure that the `module index, call index` tuples map to the
-same set of functions. It also checks if there have been any changes in `storage`. In case of a breaking change,
-increase `transaction_version`.
-
-To verify that the API did not break, manually start the following [Github
-Action](https://github.com/paritytech/cumulus/actions/workflows/extrinsic-ordering-check-from-bin.yml). It takes around
-a minute to run and will produce the report as artifact you need to manually check.
-
-When the workflow is done, click on it and download the zip artifact, inside you'll find an `output.txt` file. The
-output should be appended as comment to the "Checklist issue".
-
-The things to look for in the output are lines like:
+Offline signing libraries depend on a consistent ordering of call indices and
+functions. Compare the metadata of the current and new runtimes and ensure that
+the `module index, call index` tuples map to the same set of functions. It also checks if there have been any changes in `storage`. In case of a breaking change, increase `transaction_version`.
+
+To verify the order has not changed, manually start the following [Github Action](https://github.com/paritytech/cumulus/actions/workflows/extrinsic-ordering-check-from-bin.yml). It takes around a minute to run and will produce the report as artifact you need to manually check.
+
+To run it, in the _Run Workflow_ dropdown:
+1. **Use workflow from**: to ignore, leave `master` as default
+2. **The WebSocket url of the reference node**:
+	- Statemint: `wss://statemint-rpc.polkadot.io`
+    - Statemine: `wss://statemine-rpc.polkadot.io`
+    - Westmint: `wss://westmint-rpc.polkadot.io`
```

Comment:
@NachoPal should the rococo url also be here wss://rococo-statemint-rpc.polkadot.io 
or it is not necessary?

Reply:
no harm in it

Reply:
Why is that needed? Shouldn't be the same weights as Statemint?